client/ayon_houdini/plugins/publish/extract_render.py

```diff
@@ -58,8 +59,29 @@ def process(self, instance):
             return
 
         if creator_attribute.get("render_target") == "local":
-            ropnode = hou.node(instance.data.get("instance_node"))
-            render_rop(ropnode)
+            # FIXME Render the entire frame range if any of the AOVs does not have a
+            # previously rendered version. This situation breaks the publishing.
+            # because There will be missing frames as ROP nodes typically cannot render different
+            #  frame ranges for each AOV; they always use the same frame range for all AOVs.
+            rop_node = hou.node(instance.data.get("instance_node"))
+            frames_to_fix = clique.parse(instance.data.get("frames_to_fix", ""), "{ranges}")
+
+            if len(set(frames_to_fix)) > 1:
```

Comment:
Why would this only be for two frames and more?
```suggestion
            if len(set(frames_to_fix)) > 0:
```

Reply:
This avoids publish instances with one frame/file. e.g. HDA products, alembic products.